tests/XRootD/scitokens.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+function setup_scitokens() {
+	require_commands curl
+	if [ -z "$HOSTNAME" ]; then
+		HOSTNAME=`hostname`
+	fi
```

Comment:
This should be taken care of in [`test.sh`](https://github.com/xrootd/xrootd/blob/8a4ca755c57bd01b807a74a4aa3fc89b3f42116b/tests/XRootD/test.sh#L133). I think it's better to set hostname there for everybody. I set it for `root://` protocol already, I suggest we replace with or add
```bash
: ${HOSTNAME:=localhost}

export HOSTNAME
```
in `test.sh`, then we can reuse it [here](https://github.com/xrootd/xrootd/blob/8a4ca755c57bd01b807a74a4aa3fc89b3f42116b/tests/XRootD/http.sh#L26) as well.

Reply:
Based on your suggestions below, I took a different route -- using `localhost` across the board.  That allows some of the config files to be constant and not need to be generated at runtime.  Latest rebase removes all the `HOSTNAME` pieces.